schemas/blocks/list_collector.json

```diff
@@ -18,6 +18,18 @@
     "required": ["id", "value"],
     "additionalProperties": false
   },
+  "summary": {
```

Comment:
Should we make this key `required`?

Reply:
Asked @ajmaddaford about this yesterday - the consensus was it shouldn't be mandatory